rl.py

```diff
@@ -2,13 +2,26 @@
 This is a file that will run a simple rl model.
 """
 
+import expe
+import models
+
 def main():
```

Comment:
I prefer to minimize forward references, so would put the main function below run_rl below.